webapp/src/Doctrine/HashPasswordSubscriber.php

```diff
@@ -4,7 +4,8 @@
 
 use App\Entity\User;
 use Doctrine\Common\EventSubscriber;
-use Doctrine\Common\Persistence\Event\LifecycleEventArgs;
+use Doctrine\ORM\Event\LifecycleEventArgs;
```

Comment:
I think we’re not using this import anymore?

Reply:
We are using it in `public function preUpdate(PreUpdateEventArgs $args)`

Reply:
That is PreUpdateEventArgs, not LifecycleEventArgs?

Reply:
*We are using it in `public function prePersist(LifecycleEventArgs $args)`

Reply:
Now I see it 🤓